include/stdint.h

```diff
@@ -41,28 +41,28 @@
 
 /* Limits of exact-width integer types */
 
-#define INT8_MIN            (-INT8_MAX - 1)
-#define INT8_MAX            127
-#define UINT8_MAX           255
+#define INT8_MIN            (-INT8_MAX - INT8_C(1))
```

Comment:
should we use INT8_C(-INT8_MAX - 1) to avoid the default promotion?

Reply:
or INT8_C(-128) directly

Reply:
Sorry, I couldn't follow. Which promotion do you expect to happen in this scenario?
I don't think it is really needed, since INTx_MAX should be already suffixed (if needed by the arch) and thus it won't be promoted for after the subtraction.

Regarding `-128`, I feel that making it relative to `INT8_MAX` definition copes with code comprehension.
And also it is the same approach used by LLVM:
https://clang.llvm.org/doxygen/stdint_8h_source.html

Reply:
Actually, I'll remove the INTx_C(1) since it has no effect to the end result.

Reply:
Yes, you are right, my concern isn't valid here.